x-pack/test/accessibility/apps/snapshot_and_restore.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'snapshotRestore']);
+  const a11y = getService('a11y');
+  const es = getService('es');
+
+  describe('Stack Management - Snapshot Restore Accessibility', () => {
```

Comment:
Can we change the describe here to say 'Stack management- snapshot restore a11y tests'?  So that it says uniform with other test descriptions. Thanks!

Reply:
done